Engine update required: GeoLite2-Country.mmdb.gz cannot be downloaded

Hi,

GeoLite2-Country.mmdb.gz cannot be downloaded using the scripts in the engine repo anymore; as a result, I get the build error:

```bash
  Restore completed in 1.76 ms for /home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine/OpenRA.Mods.D2k/OpenRA.Mods.D2k.csproj.
  Restore completed in 1.48 ms for /home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine/OpenRA.Platforms.Default/OpenRA.Platforms.Default.csproj.
  Restore completed in 1.03 ms for /home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine/OpenRA.PostProcess/OpenRA.PostProcess.csproj.
  Restore completed in 1.32 ms for /home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine/OpenRA.Server/OpenRA.Server.csproj.
  Restore completed in 2.62 ms for /home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine/OpenRA.Test/OpenRA.Test.csproj.
  Restore completed in 1.68 ms for /home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine/OpenRA.Utility/OpenRA.Utility.csproj.
Updating GeoIP country database from MaxMind.
make[1]: *** [Makefile:194: geoip-dependencies] Error 6
make[1]: Leaving directory '/home/fusion809/AUR/openra-raclassic-git/src/raclassic/engine'
make: *** [Makefile:106: engine-dependencies] Error 2
```

whenever I try to build this mod. OpenRA/OpenRA#17561 fixes this error.

Thanks for your time

--- a/mod.config
+++ b/mod.config
@@ -9,7 +9,7 @@
 MOD_ID="raclassic"
 
 # The OpenRA engine version to use for this project.
-ENGINE_VERSION="release-20191117"
+ENGINE_VERSION="playtest-20200118"
 
 # .dll filenames compiled by the mod solution for use by the runtime assembly check
 WHITELISTED_MOD_ASSEMBLIES="OpenRA.Mods.Raclassic.dll"

--- a/mods/raclassic/mod.yaml
+++ b/mods/raclassic/mod.yaml
@@ -1,6 +1,6 @@
 Metadata:
 	Title: Red Alert Classic
-	Version: release-20191117
+	Version: playtest-20200118
 	Website: https://www.openra.net
 	WebIcon32: https://raw.githubusercontent.com/OpenRA/ra-classic/mods/raclassic/icon.png
 

--- a/packaging/linux/buildpackage.sh
+++ b/packaging/linux/buildpackage.sh
@@ -78,7 +78,7 @@ pushd "${ENGINE_DIRECTORY}" > /dev/null
 make clean
 
 # linux-dependencies target will trigger the lua detection script, which we don't want during packaging
-make cli-dependencies geoip-dependencies
+make cli-dependencies
 sed "s/@LIBLUA51@/liblua5.1.so.0/" thirdparty/Eluant.dll.config.in > Eluant.dll.config
 
 make core

